activiti-cloud-notifications-graphql-service/services/graphiql/src/test/resources/application.properties

```diff
@@ -3,4 +3,6 @@ keycloak.realm=activiti
 keycloak.resource=activiti
 keycloak.public-client=true
 
-server.servlet.context-path=/default-app
\ No newline at end of file
+server.servlet.context-path=/default-app
+
+activiti.cloud.notification.uri-prefix=
```

Comment:
You should not need to have to define this in test properties if there is default properties source defined for each service